dashboard/src/actions.ts

```diff
@@ -0,0 +1,145 @@
+import { Moment } from 'moment';
+
+import { RegionList, RegionStatsList } from './api/types';
+import { MapViewport } from './components/types';
+
+interface CheckExistingLoginAction {
+    type: 'CHECK_EXISTING_LOGIN';
```

Comment:
This looks really error prone, using the same string in two places instead of using a constant. Now if the interface changes, the function below needs to be updated as well. And there is most likely a reducer catching this as well and it needs to keep up to date with the string.

Reply:
The reducer won't work if it's using a different string, because then the type check will fail. I know this could use some refactoring so that the same wouldn't have to be written twice, but didn't do that yet, because it didn't occur to me right away how to do it properly (and wanted to have this PR:ed before...)

Reply:
Of course the reducer won't work, that is point 😄 To make it harder to make the mistake. But as you said, this can be refactored later if needed.

Reply:
sorry, I meant won't compile. I.e. you'll spot that error anyway